extension/main.js

```diff
@@ -13,6 +13,15 @@
 		chrome.browserAction.setTitle({
 			title: title
 		});
+
+		if (badge && window.GitHubNotify.settings.get('showDesktopNotif')) {
```

Comment:
I think it should also check if the number of notifications is higher than the previous audit: without this test, the desktop notification window is shown again and again, as long we did not read all notifications. It is painful! :-/